internal/signer/darwin/signer.go

```diff
@@ -69,7 +69,7 @@ func (k *EnterpriseCertSigner) Public(ignored struct{}, publicKey *[]byte) (err
 	return
 }
 
-// Sign signs a message by encrypting a message digest.
+// Sign signs a message digest.
```

Comment:
nit: I would take this opportunity to enhance the comments here and other places to something like:

// Sign signs a message digest specified by args and writes the output to resp.